BRPeer.c

```diff
@@ -274,6 +278,8 @@ static int _BRPeerAcceptAddrMessage(BRPeer *peer, const uint8_t *msg, size_t msg
             off += sizeof(uint16_t);
 
             if (! (p.services & SERVICES_NODE_NETWORK)) continue; // skip peers that don't carry full blocks
+
+            // TODO [HODL] Why are we ignoring IPv6?
```

Comment:
perhaps it doesn't support multihome networking, not trivial in other projects.

Reply:
@gubatron Heh it's pretty weird because I've seen it connecting to IPv6 nodes on logs by users (like @docbtc) Me personally I don't know why not support them...